src/protections/BasicFlooding.ts

```diff
@@ -28,6 +28,8 @@ export class BasicFlooding implements IProtection {
     private lastEvents: { [roomId: string]: { [userId: string]: { originServerTs: number, eventId: string }[] } } = {};
     private recentlyBanned: string[] = [];
 
+    settings = {maxPerMinute: 10};
```

Comment:
Documentation needed, as usual :)

I assume that the next step is actually using these settings, right?